### ISSUE TITLE
Project creation issues

### ISSUE THREAD

**Issue by bmatzner** [dave-redfern/laravel-doctrine-project#1]
Hi Dave,

thanks for sharing your project template. Awesome!

I ran into two issues on a mac with PHP 7.1

When first creating the project with composer, the vendor packages are not installed first, so

`php artisan clear-compiled`

runs into

```
Fatal error: require(): Failed opening required '[PROJECT]/bootstrap/../vendor/autoload.php' (include_path='.:') in [PROJECT]/bootstrap/autoload.php on line 17`
```
This can be worked around by temporarily removing the pre-update-cmd and then installing by hand.

Thereafter, however, this occurs:

```
Script php artisan clear-compiled handling the pre-update-cmd event returned with error code 1:
 
 [Illuminate\Contracts\Container\BindingResolutionException]
  Target [Doctrine\Common\Annotations\Reader] is not instantiable while build
  ing [LaravelDoctrine\ACL\Mappings\Subscribers\BelongsToOrganisationsSubscri
  ber].
```

**Comment by dave-redfern** [comment]
@bmatzner Hi! My apologies for missing your issue. I'll work on addressing it now.

**Comment by dave-redfern** [comment]
Fixed that autoload issue with 3.1.0, however there remains the problem that the artisan command tries to connect to the (un-configured) db because Doctrine is creating a connection and it fails.
